Newsletters: add 261 (2023-07-26) (#1211)
bitcoinops/bitcoinops.github.io: _topics/en/mempool-clustering.md

---

> title: Mempool clustering

Reviewer: At least from what I’ve been hearing the byword I’ve been hearing for this is:

```suggestion
title: Cluster Mempool
```

  - Mining

## Optional.  Produces a Markdown link with either "[title][]" or
## "[title](link)"
primary_sources:
    - title: "Bitcoin Core #27677: new mempool design"
      link: https://github.com/bitcoin/bitcoin/issues/27677

## Optional.  Each entry requires "title" and "url".  May also use "feature:
## true" to bold entry and "date"
optech_mentions:
  - title: "Bitcoin Core meeting transcript about mempool redesign"
    url: /en/newsletters/2023/05/17/#mempool-clustering

  - title: "Mempool proposals, including mempool clustering"
    url: /en/blog/waiting-for-confirmation/#policy-proposals
    date: 2023-07-12

# FIXME: this week's mention

## Optional.  Same format as "primary_sources" above
see_also:
  - title: Package relay
    link: topic package relay

  - title: Replace-by-Fee
    link: topic rbf

## Optional.  Force the display (true) or non-display (false) of stub
## topic notice.  Default is to display if the page.content is below a
## threshold word count
#stub: false

## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
## Should be less than 500 characters
excerpt: >
  **Mempool clustering** is a proposal to associate each unconfirmed

>   transaction in a mempool with any related transactions, creating a

Reviewer 2: ```suggestion
  transaction in a mempool with all related transactions, creating a
```

  cluster.  Each cluster of transactions, whether it be a single
  transaction or several transactions, can be ordered from most
  desirable to mine to least desirable, allowing operations for adding
  or removing new clusters to complete fast enough to use them in P2P
  network code.

---
If P2P network code can determine whether a new unconfirmed transaction
(or cluster of transactions) will improve the profitability of the
mempool for miners in the near term, that criteria can be used instead
of other heuristics for determining when a transaction [package][topic
package relay] or [replacement][topic rbf] should be accepted.  It is

> hoped that this can allow transaction relay to provide more flexible

Reviewer 2: ```suggestion
hoped that this allows transaction relay to provide more flexible
```
